### PR TITLE
Update to 1.0.1

### PR DESCRIPTION
**Pull request by ericpre**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

Close #16.

### CODE REVIEW

**Comment by ericpre** [comment]
@conda-forge/prettytable, this PR is ready for review.

**Comment by ericpre** [comment]
cc @hugovk.

**Comment by hugovk** [comment]
I'm not very familiar with conda-forge, but the updated URL and dependencies look right 👍

**Comment by ericpre** [comment]
Ping @conda-forge/prettytable, could you please review this PR?

**Comment by ericpre** [comment]
@conda-forge/core, would it be possible to get this PR merged?

From a quick search on github of the maintainer of this recipe (https://github.com/orgs/conda-forge/teams/prettytable/members), they don't seem to be active anymore on conda-forge. I have added myself to help with the maintenance of this recipe. 

@hugovk, would you like to be added as recipe maintainer too, since you are now maintaining prettytable? It would be useful if you could follow what is going on with this recipe!

**Comment by chrisburr** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
@conda-forge-admin please rerender

**Comment by ocefpaf** [comment]
Oops... @chrisburr was faster than me :smile:

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,33 +1,29 @@
...
-  number: {{ build }}
-  script: python -m pip install --no-deps --ignore-installed .
-
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - pip
     - python
```
Comment (chrisburr) [inline recipe/meta.yaml:21]: This should match run:
```suggestion
    - python >=3.5
```

**Comment by chrisburr** [comment]
> Oops... @chrisburr was faster than me 😄

You got a full commit in before I commented!

**Comment by ericpre** [comment]
Thanks!

**Comment by hugovk** [comment]
I'm not very familiar with Conda so not sure I'd be able to maintain this recipe. I am watching this repo though, and will comment if there's anything I spot. For example:

PrettyTable 1.0.1 actually supports 2.7 and 3.5+, but fine if this recipe is only for 3.5+. It's very unlikely we'd fix anything for Python 2.

The next PrettyTable release (probably 2.0.0) will be 3.6+. The EOL versions are already dropped in `master`.

**Comment by ocefpaf** [comment]
> I'm not very familiar with Conda so not sure I'd be able to maintain this recipe. I am watching this repo though, and will comment if there's anything I spot. For example:
> 
> PrettyTable 1.0.1 actually supports 2.7 and 3.5+, but fine if this recipe is only for 3.5+. It's very unlikely we'd fix anything for Python 2.
> 
> The next PrettyTable release (probably 2.0.0) will be 3.6+. The EOL versions are already dropped in `master`.

@hugovk thanks! That kind of info is why having authors as maintainers is important. You don't need to worry about the conda details if you can every-now-and-them make comments like that. At the moment we are not building 2.7 and, while it would be nice to have it there for completeness, it won't get built. We need to watch out for the next release and update the Python constraint.

**Pull request merged**